src/eventDispatchers/mouseToolEventDispatcher.js

```diff
@@ -303,14 +338,36 @@ function mouseDownActivate (evt) {
 }
 
 function mouseDoubleClick (evt) {
+  if (isAwaitingMouseUp) {
+    return;
+  }
   console.warn('mouseDoubleClick');
+
+  let tools;
+  const eventData = evt.detail;
+  const element = eventData.element;
+
+  // Filter out disabled, enabled, and passive
+  tools = getActiveToolsForElement(element, getters.mouseTools());
+  tools = tools.filter((tool) =>
+    isMouseButtonEnabled(eventData.which, tool.options.mouseButtonMask)
+  );
+  tools = tools.filter((tool) => typeof tool.mouseDoublClickCallback === 'function');
```

Comment:
mouseDoublClickCallback - missing an 'e'?

Reply:
Hah. Good callout

Reply:
Oops, this bit isn't used yet. You saved the annotation tool @swederik ! Will change tomorrow.